mod_ci/controllers.py

```diff
@@ -442,6 +442,50 @@ def kvm_processor(db, kvm_name, platform, repository, delay):
             platform=platform, id=test.id))
 
 
+def ci_badge_windows(status, platform, test_type, branch="master"):
+    platform = test.platform
+    if platform is TestPlatform.linux:
+        if test_type == TestType.commit:
+            if state == Status.SUCCESS:
+                with open('../static/badges/Windows_success.svg',
+                          'r') as svg:
+                    print svg.read()
```

Comment:
Printing it is just as useless as returning it. I'm unsure why you're doing that?

Reply:
so I don't need them?
just read them and then leave right?

Reply:
And what does reading them achieve? They need to be displayable if you access a fixed url for that platform (see Saurabh's comment from yesterday)

Reply:
reading them only makes them well .. read
i am confused what to use to display them

Reply:
.show()?

Reply:
This is going nowhere like this. Please take some time to read the comment @saurabhshri  posted yesterday, and try to derive a solution from that comment.

Reply:
oh shutil